lang/java/avro/src/main/java/org/apache/avro/reflect/ReflectData.java

```diff
@@ -877,6 +881,18 @@ private void consumeAvroAliasAnnotation(Class<?> c, Schema schema) {
     }
   }
 
+
+  private void consumeFieldAlias(Field field, Schema.Field recordField) {
+    if (field.isAnnotationPresent(AvroAlias.class)) {
```

Comment:
Field.isAnnotationPresent(Class) checks whether Field.getAnnotation(Class) returns null. I would suggest checking for alias != null to avoid the double invocation.